### PR TITLE
ldpl: upstream Linux fix

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by iMichka — APPROVED** [review]

**Review by iMichka — CHANGES_REQUESTED** [review]
Is man-db provided by macOS?

File: Formula/ldpl.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ class Ldpl < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "01f2a987ba4b74d1b50374c7a9a616703a2a8ad479aaad8b80ed8e936af91d80"
   end
 
+  on_linux do
+    depends_on "man_db"
```
Comment (dawidd6) [inline Formula/ldpl.rb:17]:
```suggestion
    depends_on "man-db"
```

**Comment by danielnachun** [comment]
It looks `man` is provided by macOS and the `man-db` formula even prepends `g` to the `man` to avoid conflicting with the macOS version.  I don't why this isn't already in the list of formula for `uses_from_macos` but I'm going to add it.

**Comment by iMichka** [comment]
It's weird that it's only needed on linux but not on Mac then. It sometimes happens, but it's quite rare. We try to keep the dependencies aligned as much as possible. Can you run a build on linux and see what fails if we remove that dependency?

**Comment by danielnachun** [comment]
Oh I forgot to add that I tried that in Docker.  The `make install` fails on Linux, because it expects to find the `man` command.

**Comment by iMichka** [comment]
I see. `man-db` could be made `uses_from_macos` but it is not equivalent to `man` on Mac, as the package installs way more things. So I'm fine with this change, let's keep it as-is.

**Comment by iMichka** [comment]
Please change to `depends_on "man-db"`, and this should be good to go.

**Review by iMichka — APPROVED** [review]

**Pull request merged**